cypress/integration/articleSpec.js

```diff
@@ -89,4 +89,22 @@ describe('News Article', () => {
       .should('have.attr', 'src')
       .and('match', /(\/static\/js\/bundle\.\w+\.js)/g);
   });
+
+  it('should check the font file loads', () => {
```

Comment:
We should change this to be more specific to what is being asserted in the test, e.g. `should load less than 3 fonts`